About.html

```diff
@@ -0,0 +1,115 @@
+                    <li><a href="#" class="item-join">Join</a></li>
+                    <li><a href="#" class="item-sponsor">Sponsor</a></li>
+                    <li><a href="#" class="item-newa">News</a></li>
+                   
+                </ul>
+                </nav>
+            <button class="campaign">
+                Referral program
+            </button>
+        </div>
+    </header>
+    <section class="about">
+        <div class="small-about">
+        <h3>"Join the Musical Celebration, Experience Unity!"</h3>
+        <h1>HARMONY IN MOTION 2023</h1>
+        <p class="description">Experience the magic of music like never before with 'Harmony in Motion 2023'. This incredible event is set to captivate your senses and transport you to a world where rhythm and melody reign supreme. Whether you're a fan of soulful ballads or high-energy beats, our talented artists will take you on a musical journey that you won't forget. With stunning visuals, electrifying performances, and an ambiance that exudes harmony, this is one event that you simply cannot miss. So, mark your calendars and get ready to immerse yourself in the sound of 'Harmony in Motion 2023'!</p>
+        <p class="date">
+            We are more than happy to assist you in any way we can. Thank you for your interest in "Harmony in Motion 2023" and we hope to see you there! For any further questions, please feel free to contact us at <br><a href="#">harmonyinmotion2023@gmail.com</a>.</p>
+        </div>
+        </section>
+    <section class="logo">
+        <h3>Harmony In Motion 2023</h3>
+        <hr>
+        <p></p>
+        <img src="Images/Harmony in motion logo.png">
+    </section>
+    <section class="past">
+        <div class="small-past">
+        <h2>The World as Our Stage: A Look Back at Harmony in Motion's Unforgettable Performances</h2>
+        <hr>
+        <p>Welcome to Harmony in Motion 2023! As we prepare for this year's musical extravaganza, let's take a moment to reflect on the past events. Harmony in Motion has brought together music lovers from around the world, creating a space of unity and rhythm. From the lively streets of New Orleans, the scenic beaches of Rio de Janeiro, the vibrant culture of Nigeria, to the bustling energy of London, Harmony in Motion has left a lasting impact on each city it visited. To get a glimpse of the unforgettable moments from our past events, check out below.</p>
+    </div>
+        <div class="images">
+            <div class="image1">
+                <img src="Images/concert1.jpg" alt="concert1">
+                <div class="overlay">
+                <h3>2020</h3>
+                <p>Harmony In Motion 2020 in Rio de Janeiro</p>
+            </div>
+        </div>
+            <div class="image2">
+            <img src="Images/concert3.jpg" alt="concert3">
+            <div class="overlay">
+            <h3>2018</h3>
+            <p>Harmony In Motion 2018 in New Orleans</p>
+        </div>
```

Comment:
- [ ] Good job working on the past section, but in the original design there are only two cards but yours have three. Please kindly include only two cards as specified in the design.
- [ ] Also, the original design cards do not have a border-radius, please kindly ensure that your matches it.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219871813-974b8e93-31ba-4aae-a7d1-3094f44e8c4e.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219871841-96d52750-3f3f-487f-9204-f65b39fb088a.png)